Rewrite of release schedule FAQ (#15151)
home-assistant/home-assistant.github.io: source/_faq/release.markdown

---
title: "Releases"
description: "Home Assistant release schedule"
ha_category: Common
---

> New major versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 

Reviewer: Actually, minor, as we have not reached 1.0 yet.

Author: ```suggestion
New versions of Home Assistant are released every three weeks on a Wednesday. The exact dates can be seen in the upcoming events calendar on the [Home Assistant Developers](https://developers.home-assistant.io/) website. 
```
My intention with "major" was to try and differentiate between feature releases and hotfixes. But in hindsight that doesn't belong here. I'll remove the word completely.

> This release schedule allows for two weeks of development time and one week of beta testing. Users who participate in the beta can view the changelog under [beta release notes](https://rc.home-assistant.io/latest-release-notes/) and get help in the `#beta` channel of [Home Assistant's Discord server](https://discord.gg/RMC8WkX). Testers are also encouraged to [report issues on GitHub](/help/#bugs-feature-requests-and-alike).

Reviewer: This is not correct. We have a 3 week development cycle. With the release of the beta, the development of the next release starts as well.

Reviewer: use `/join-chat` for linking to our Discord.

Author: ```suggestion
The third week of our release schedule is primarily focused on beta testing. Users who participate in the beta can view the changelog under [beta release notes](https://rc.home-assistant.io/latest-release-notes/) and get help in the `#beta` channel of [Home Assistant's Discord server](/join-chat). Testers are also encouraged to [report issues on GitHub](/help/#bugs-feature-requests-and-alike).
```
Maybe this is better?
